lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -48,6 +48,12 @@ def setup_rabbitmq_connection_config
         # Time in seconds to wait before retrying a connection
         config :connect_retry_interval, :validate => :number, :default => 1
 
+        # The default connection timeout; zero means wait indefinitely
+        config :connection_timeout, :validate => :number
```

Comment:
I think the zero makes sense here actually. I think that if unspecified it should be a sane default (which in this case is set by March Hare). Unfortunately, AFAIK, there's no way in logstash to set a value to nil.

Given that we should keep the current behavior. I probably should make the default value explicit here, however, to aid the docs.

What are your thoughts @jordansissel ?